README.md

```diff
@@ -59,7 +75,7 @@ There are several branches in the repository.  Here are the ones that you should
  -  [Tomcat 7+] (http://tomcat.apache.org/download-70.cgi)
     Apache Tomcat is an open source software implementation of the Java Servlet and JavaServer Pages technologies.
 
- -  [Node v12+] (https://nodejs.org/) 
+ -  [Node v12+] (https://nodejs.org/)
```

Comment:
Is it really still Tomcat 7?  I thought we'd moved to 8.  Also, please check all the other software version statements and the "supported browsers" statement.